modules/custom/fdi_default_content/fdi_default_content.module

```diff
@@ -38,3 +38,51 @@ function fdi_default_content_form_alter(&$form, &$form_state, $form_id) {
     }
   }
 }
+
+/**
+ * Implements hook_mail().
+ */
+function fdi_default_content_mail($key, &$message, $params) {
+
+  switch ($key) {
+    case 'insert_contacto_node':
+      $message['from'] = \Drupal::config('system.site')->get('mail');
+      $message['subject'] = $params['message_subject'];
+      $message['body']['message_intro'] = $params['message_intro'];
+      $message['body']['nombre'] = 'Nombre: ' . $params['entity']->get('title')->getValue()[0]['value'];
```

Comment:
Can we please have the label in a different line?

Nombre:
[NOMBRE]

Correo electrónico:
[CORREO]

etc...